inf-clojure.el

```diff
@@ -285,10 +285,11 @@ buffers after they are created with `rename-buffer'."
            (inf-clojure-repl-p))
       (setq inf-clojure-buffer (current-buffer))
     (let ((repl-buffers (inf-clojure-repls)))
-     (if (> (length repl-buffers) 0)
-         (when-let ((repl-buffer (completing-read "Select default REPL: " repl-buffers nil t)))
-           (setq inf-clojure-buffer (get-buffer repl-buffer)))
-       (user-error "No buffers have an inf-clojure process")))))
+      (if (> (length repl-buffers) 0)
+          (when-let ((repl-buffer (completing-read "Select default REPL: " repl-buffers nil t)))
+            (setq inf-clojure-buffer (get-buffer repl-buffer)))
+        (user-error "No buffers have an inf-clojure process"))))
+  (message "Current inf-clojure REPL set to %s" inf-clojure-buffer))
```

Comment:
Also, should this `message` move this to be inside of the `when-let` which sets `inf-clojure-buffer`?

Reply:
You're right, I missed that possible outcome.
I changed it to only display the message after a successful change. Not confident in my style choices here, I could also just repeat the call to `message` in both success branches separately, let me know if you have a preference. 
https://github.com/jjttjj/inf-clojure/blob/04b967f1e60adb79cd84d580268f7af00021eb71/inf-clojure.el#L284-L294

Reply:
I don't have any strong opinions. I lean towards having the `message` and the `setq` in the same body so they work in tandem. But it doesn't really matter.